app/services/authentication/providers.rb

```diff
@@ -44,6 +44,8 @@ def self.available?(provider_name)
     # TODO: [@forem/oss] ideally this should be "available - disabled"
     # we can get there once we have feature flags
     def self.enabled
+      return [] if SiteConfig.invite_only_mode
```

Comment:
This is probably the most important part of the change.

Reply:
Can we add a specific test for `.enabled` ? see the spec for this file